style.css

```diff
@@ -464,3 +492,370 @@ textarea {
+    padding-bottom: 1rem;
+  }
+
+  .tonic {
+    display: none;
+  }
+
+  .deskpar1 {
+    margin-left: -18.3rem;
+    margin-top: 11.5rem;
+    width: 38%;
+  }
+
+  .deskpar2 {
+    width: 38%;
+  }
+
+  .deskpar4 {
+    width: 40%;
+    margin-top: 12.4rem;
+    margin-left: -24%;
+  }
+
+  .deskpar5 {
+    width: 40%;
+    margin-left: 6%;
+  }
+
+  .deskparY {
+    display: none;
+  }
+
+  .deskpar3,
+  .deskpar4,
+  .deskpar5 {
+    display: flex;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    color: rgba(52, 69, 99, 1);
+  }
+
+  .deskpar3 {
+    width: 40%;
+    margin-left: 6%;
+  }
+
+  .languages-1 {
+    margin-left: 12rem;
+    margin-top: -15.5rem;
+  }
+
+  .languages-2 {
+    margin-left: -36rem;
+    margin-top: -16rem;
+  }
+
+  .languages-3 {
+    margin-left: 20rem;
+    margin-top: -13rem;
+  }
+
+  .project-1 {
+    margin-left: 5rem;
+    margin-top: -10rem;
+  }
+
+  .project-2 {
+    margin-right: 71%;
+    margin-top: -11rem;
+  }
+
+  .project-3 {
+    margin-left: 5rem;
+  }
+
+  .project-4 {
+    margin-right: 71%;
+    margin-top: -10em;
+  }
+
+  #about-self {
+    background-image: url(Assets/files/about\ myself\ -\ background-\ shape-\ desktop.jpg);
+    border-top-right-radius: 10rem;
+    width: 1330px;
+    height: 916px;
+    margin-top: 10rem;
+    margin-left: 0.3rem;
+  }
+
+  .aboutself-heading {
+    margin-left: 8%;
+  }
+
+  .aboutself-paragragh {
+    margin-left: 8%;
+    width: 33%;
+  }
+
+  .connect2 {
+    margin-left: 8%;
+  }
+
+  #social-icons {
+    margin-left: 5.5%;
+  }
+
+  .resume {
+    margin-left: 8%;
+    margin-top: 2rem;
+  }
+
+  .languages1 {
+    margin-left: 48%;
+    margin-top: -28%;
+  }
+
+  #tech {
+    display: flex;
+  }
+
+  .tag {
+    width: 118px;
+    height: 118px;
+    flex-direction: column-reverse;
+    margin-right: 4.5rem;
+    background-position-y: 15%;
+    background-position-x: 20%;
+    padding-top: 8%;
+    padding-right: 2px;
+  }
+
+  .tag2 {
+    padding-right: 2rem;
+  }
+
+  .tag3 {
+    padding-right: 3.5rem;
+  }
+
+  .divider {
+    display: flex;
+    width: 488px;
+    margin-left: 44%;
+    margin-top: 3%;
+  }
+
+  .frameworks {
+    margin-left: 45.5%;
+  }
+
+  .skills {
+    margin-left: 45.5%;
+  }
+
+  .arrow {
+    margin-left: 60%;
+  }
+
+  .arrow1,
+  .arrow2 {
+    margin-left: 12rem;
+  }
+
+  .footer {
+    background-color: rgba(96, 112, 255, 1);
+    width: 100%;
+    height: 846px;
+    margin-top: -10rem;
+  }
+
+  .footer-container {
+    background-image: url(images/files/footer-background-desktop.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    align-items: center;
+  }
+
+  .footer-heading {
+    padding-top: 12%;
+    padding-bottom: 1rem;
+  }
+
+  .footer-paragraph {
+    height: 124px;
+    width: 540px;
+    margin-bottom: -3rem;
+  }
+
+  .flex {
+    width: 35%;
+    align-items: center;
+    padding-left: 32%;
+  }
+}
```

Comment:
- You have done an amazing job with this submission :clap: :clap: but then there are a few changes that would be worth making in order to ensure you have a well-responsive portfolio website for all devices. I have recorded this [video](https://watch.screencastify.com/v/bGq1BBWbQdJ5uIuqD2Wb) to act as a guide as per my experience on your website. We have some spacing imbalances, non-responsive images, and other elements like `headings`, `paragraphs`, `divs`, etc. 
- [ ] Kindly look into the above and see how you can make the design better.